docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -88,7 +85,41 @@ gradient boosting methodologies.
 //end::regression-algorithms[]
 
 
-[discrete]
+[[dfa-regression-lossfunction]]
+===== Loss functions for {regression} analyses
+
+Loss function is for measuring how well a given {ml} model fits the specific 
+data set. The loss function boils down all the different aspects of the model to 
+a single number. The bigger the difference between the prediction and the ground 
```

Comment:
```suggestion
Loss function measures how well a given {ml} model fits the specific 
data set. The loss function boils down all the different under- and overestimations of the model to 
a single number. The bigger the difference between the prediction and the ground 
```

Reply:
Or something along those lines. Strictly speaking, loss function doesn't care about the properties/aspects of the model, it only takes the predictions and the ground truth into account.